packages/core/src/Search.cpp

```diff
@@ -24,6 +24,16 @@
 
 #include <cstdlib>
 
+#ifdef WIN32
+#define ENV_LIB_PATH "CMAKE_PREFIX_PATH"
```

Comment:
I suspect a better Windows analogue to `LD_LIBRARY_PATH` might be `CMAKE_PREFIX_PATH/lib`. Then we wouldn't need `LIB_PATH` anymore; we could just change that back to `"soss"`.

That being said, feel free to correct me if there's some other reason that Windows should use `lib/soss` in the lookup scheme whereas Linux systems would just use `soss`.

Reply:
Thanks for the advice! I've been thinking about this and I think is a little bit complicated. When you source SOSS `CMAKE_PREFIX_PATH` stores all directories for each package. I understand that this is done using colcon templates so is impossible to set directories inside `CMAKE_PREFIX_PATH/lib` without changin colcon templates. In addition, mix files are being generated in `install\<package>\lib`. Correct me if I'm understanding wrong the workflow but I thougth adding lib to `CMAKE_PREFIX_PATH` was very easy solution because that variable stores all install directories. Unfortunately, this means that, for Windows, we have to add lib to the path but, in the other hand, users will be more confortable with just `CMAKE_PREFIX_PATH` variable if they decide to not use colcon.

Reply:
Oh right, sorry, I didn't have my head on straight when I made that comment. Please disregard.